overrides/home.html

```diff
@@ -236,6 +236,20 @@ <h1>
             </p>
           </a>
         </div>
+        <div class="col">
+          <a href="{{ 'about/case-studies/ibm/' | url }}">
+            <div class="logo">
+              <img class="ibm" src="images/case-studies/ibm.png" alt="IBM logo" draggable="false" width="75" />
+            </div>
+            <hr>
+            <p class="normal-text">
+              "The introduction of the ML training infrastructure using Knative Eventing 
+              has enabled us to establish a well-defined operational boundary for the service teams."<br><br>
+
+              —IBM
+            </p>
+          </a>
+        </div>
```

Comment:
Let's only show 3 case studies in home page.

Can you check the dates of the case studies and update the homepage to show the latest 3?

Reply:
@aliok The existing three case studies are not available on the [CNCF website](https://www.cncf.io/case-studies?_sft_lf-project=knative). Could you please tell me where these case studies were published?

I have verified that the [PR](https://github.com/knative/docs/pull/4406) to add these was merged in 2021, so all four case studies on the CNCF website are newer than these three.

Reply:
You're right.

Then can you identify the latest ones from the PR dates? Let's show the latest 3 only.